src/schemas/complaint-tool/schema.js

```diff
@@ -0,0 +1,165 @@
+import definitions from '../../common/definitions';
+import schemaHelpers from '../../common/schema-helpers';
+import _ from 'lodash';
+
+let schema = {
+  $schema: 'http://json-schema.org/draft-04/schema#',
+  title: 'GI BILL SCHOOL COMPLAINT TOOL',
+  type: 'object',
+  additionalProperties: false,
+  definitions: {
+    fullName: _.merge({}, definitions.fullName, { // TODO: our common definition includes "II" (inclusion pending stakeholder feedback), lacks "Other"
+      properties: { // First, Middle, Last (100 limit, each)
+        prefix: {
+          type: 'string',
+          'enum': [
+            'Mr.',
+            'Mrs.',
+            'Ms.',
+            'Dr.',
+            'Other'
+          ]
+        },
+        suffix: {
+          type: 'string',
+          'enum': ['Other']
+        }
+      }
+    })
+  },
+  required: [ // no fields are required for submission, though several are required by design on FE
+    // 'onBehalfOf',
+    // 'educationDetails',
+    // 'issue',
+    // 'issueDescription',
+    // 'issueResolution'
+  ],
+  properties: {
+    onBehalfOf: {  // Type: text (255 limit)
+      type: 'string',
+      'enum': [
+        'Myself',
+        'Someone else',
+        'Anonymous'
+      ]
+    },
+    serviceBranch: { // Type: text (255 limit)
+      type: 'string',
+      'enum': [
+        'Army',
+        'Navy',
+        'Marines',
+        'Air Force',
+        'Coast Guard',
+        'NOAA/PHS'
+      ]
+    },
+    serviceAffiliation: { // Type: text (255 limit)
+      type: 'string',
+      'enum': [
+        'Service Member',
+        'Spouse or Family Member',
+        'Veteran',
+        'Other'
+      ]
+    },
+    fullName: {
+      '$ref': '#/definitions/fullName'
+    },
+    email: {
+      type: 'string',  // Type: email (no length limit)
+      format: 'email'
+    },
+    educationDetails: {
+      required: [ // no fields are required for submission, though several are required by design on FE
+        // 'school',
+        // 'programs'
+      ],
+      school: definitions.school,
+      programs: { // TODO: Needs to be translated into a (comma) delimited string, and spelled out if within 255 limit 
+        type: 'object',
```

Comment:
Maybe could use an array of enum here.

Reply:
Thanks @rian, these will be presented as checkboxes, which we typically create using booleans.